.azure-pipelines/gpu-tests.yml

```diff
@@ -29,10 +29,10 @@ jobs:
     container:
       # base ML image: mcr.microsoft.com/azureml/openmpi3.1.2-cuda10.2-cudnn8-ubuntu18.04
       # run on torch 1.8 as it's the LTS version
-      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
+      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.8"
```

Comment:
We were using 3.7 because it's the version used by most of our users.

Can we keep 3.7?

Reply:
```suggestion
      image: "pytorchlightning/pytorch_lightning:base-cuda-py3.7-torch1.8"
```